lxd/api_1.0.go

```diff
@@ -343,6 +344,11 @@ func api10Get(d *Daemon, r *http.Request) response.Response {
 		} else {
 			env.DriverVersion = driver.Info.Version
 		}
+
+		// Indicate virtualization support.
+		if driver.Info.Type == instancetype.VM {
+			env.Virtualization = true
```

Comment:
Perhaps we should add a slice of supported instance types instead.

Reply:
Sure I can change that. I see a benefit in this if we ever support more instance types (microVMs or something similar).

Should we have a field `supported_instance_types` instead?
```
supported_instance_types: "container | virtual-machine"
```

Reply:
Yeah, how about `instance_types:` and we certainly dont want to go down the path of using pipe delimited strings again like the other fields, instead we should use a slice of strings with each element containing a supported instance type name.

Reply:
I like that. Thanks for suggestion.

Reply:
Cool. It is possible to have a system that supports VMs but not containers, so keeping it generic helps.